contents/blog/product-engineer-vs-product-manager.md

```diff
@@ -0,0 +1,75 @@
+
+### Building the product
+
+Once product engineers and managers get a basic idea of what to work on, their paths begin to diverge. Product engineers dive quickly into building. They get their hands dirty writing code. They prototype and experiment. They ship previews, MVPs, and betas of features to get hands-on, direct feedback on the product.
+
+Product managers take more time before handing off work to engineers. They do further research into the business case for a product and understand where it aligns in the competitive landscape. They also spend time gaining depth into what the needs of users are and understanding their business and technology stack. 
+
+They use all this information to craft plans, requirements, and specs that solve the user needs they identified. They must translate this into something the whole team can understand. Product engineers face a similar task but have a smaller group to communicate with. Both roles require communication skills, but product managers place a higher emphasis on them.
+
+Once product managers completed their analysis and plans, they manage a team of "stakeholders" to get a product shipped. This can include design, marketing, ops, engineering, and more. They work with all of these teams to get the feature developed and launched. This ensures alignment across teams and lowers the risk of potential issues.
+
+> **Sidenote:** If you hear the word "stakeholders," you are likely talking to a product manager. They deal with many, internally and externally.
+
+Product engineers, on the other hand, ship on their own. They figure out requirements, designs, infrastructure needs, and implementation details. They rely on their broad set of technical skills for this. They may need help in design, DevOps, or infrastructure, but mostly act like a pack of wolves in search of product market fit.
+
+### Launching and beyond
+
+Launching a product is a big deal for product managers. They again work with their stakeholders to make sure it goes smoothly. Product engineers are shipping so often that a single launch doesn’t matter as much to them. They continuously deploy and use feature flags to ensure their launches go smoothly. Product managers are hands-on with launches, while product engineers automate them as much as possible.
+
+Both roles are responsible for their product after launch. This means listening to feedback and monitoring analytics. Product engineers ship bug fixes and tweaks, while product managers begin their development cycle again.
+
+## What does the rise of product engineers mean to product managers?
+
+Product engineers are taking on many of the responsibilities product managers once had, especially at the early-stage. By making product decisions and shipping code, product engineers can better help companies find product market fit. This often requires rapid experimentation rather than research and planning.
```

Comment:
```suggestion
Product engineers are taking on many of the responsibilities product managers once had, especially at early-stage companies. By making product decisions and shipping code, product engineers can better help companies find product-market fit. This often requires rapid experimentation rather than research and planning.
```